libraries/ui-library/src/components/six-item-picker/six-item-picker.tsx

```diff
@@ -413,7 +413,7 @@ export class SixItemPicker {
   }
 
   private isNextNumberDisabled() {
-    return Number(this.value) + this.step > (this.max ?? Infinity);
```

Comment:
`Number(this.max)` will always be either a number or `NaN` making `?? Infinity` pointless.

Reply:
You are right. But how should we then handle that case? 

We could first check if the value is a number and otherwise just set the infinity value:

```
let maxValue = Number.isNaN(Number(max)) ? Infinity : Number(max);
```